src/utils/components/ListPageFilter/ListPageFilter.tsx

```diff
@@ -83,12 +90,38 @@ const ListPageFilter: FC<ListPageFilterProps> = ({
 
   const textFilters = useSearchFiltersParameters(searchFilters);
 
-  const alreadySearchedParam = searchFilters?.find(
-    (searchFilter) => textFilters[searchFilter.type],
-  )?.type;
+  const getSearchTextPlaceholder = (searchType: string) => {
+    if (searchType === STATIC_SEARCH_FILTERS.name)
+      return nameFilterPlaceholder || STATIC_SEARCH_FILTERS_PLACEHOLDERS.name;
+
+    return (
+      STATIC_SEARCH_FILTERS_PLACEHOLDERS[searchType] ||
+      t('Search by {{filterName}}...', {
+        filterName: selectedSearchFilter?.filterGroupName,
+      })
+    );
+  };
+
+  const filterDropdownItems: Record<string, string> = {
+    ...searchFilters.reduce(
+      (acc, filter) => ({
+        ...acc,
+        [filter.type]: filter.filterGroupName,
+      }),
+      {},
+    ),
+  };
+
+  if (!hideLabelFilter && !hideNameLabelFilters) {
+    filterDropdownItems.labels = t('Label');
+  }
+
+  if (!hideNameLabelFilters) {
+    filterDropdownItems.name = t('Name');
+  }
```

Comment:
why not adding it to filterDropdownItems ? it as a reduce funtion

Reply:
name and label are not part of `searchFilters`. Name and label are added by default no matter the `searchFilters` state. To remove them you have to use `hideNameLabelFilters` and `hideLabelFilter`